capellambse_context_diagrams/_elkjs.py

```diff
@@ -85,8 +84,14 @@
     "widthApproximation.targetWidth": 1,  # width / height
     "elk.contentAlignment": "V_TOP H_CENTER",
 }
-LABEL_LAYOUT_OPTIONS = {"nodeLabels.placement": "OUTSIDE, V_BOTTOM, H_CENTER"}
+LABEL_LAYOUT_OPTIONS: LayoutOptions = {
```

Comment:
I think this isn't being used anywhere anymore and was possibly made obsolete by makers.SYMBOL_LAYOUT_OPTIONS.

Reply:
Yes. We can remove it.